spec/fake_braintree/subscription_spec.rb

```diff
@@ -88,6 +88,12 @@
     end
   end
 
+  it "sets the creation time" do
+    subscription = create_subscription.subscription
+
+    creation_time = Time.parse(subscription.created_at)
+    expect(creation_time).to be_within(1).of(Time.now)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.